ui/framework/src/ui-framework/UserInfo.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** Information on the authenticated user.
+ * @beta
+ */
+export class UserInfo {
```

Comment:
if youre using this internally in ui-frameowrk now, might want to export it.
but prob want to make sure this is even necessary in UiFramework

Reply:
@pbell97 is this still actually used anywhere in ui-framework or is it not needed now that we removed the SignIn/SignOut components?

Reply:
@calebmshafer So it looks like it is being used in the actual UiFramework class as well as a few other small places like SessionState. Backstage (which is deprecated) has it passed it but doesn't look like its actually being used in that component that I can tell. 
If you think it's not needed in Ui-Framework then we could pull it out (delete UiFramework.setUserInfo & getUserInfo). Regardless, it looks like its being used pretty deeply in one or two components in ui-test-app, so we would want to move it into there.

Reply:
Lets punt on removing this for the current PR. We have other changes riding on this one. I would like to follow up with this before closing breaking changes though.